--- a/configure	2019-08-30 09:46:56.000000000 -0700
+++ b/configure	2019-12-08 16:17:05.000000000 -0800
@@ -9886,6 +9886,11 @@
         arm*-*)
             CFLAGS_FOR_GOTOS="$CFLAGS_FOR_GOTOS -fomit-frame-pointer"
             ;;
+        aarch64-*)
+            echo "gcc labels test does not work on aarch64; skipping..." 1>&6
+            mercury_cv_asm_labels=no
+            mercury_cv_gcc_labels=no
+            ;;
         # On hppa64, ia64 and powerpc64 the test programs appear to work, but
         # larger programs die with an Illegal Instruction fault.
         # On powerpc (POWER) the test program enters an infinite loop;
